esphome/components/max31855/max31855.cpp

```diff
@@ -22,63 +23,84 @@ void MAX31855Sensor::setup() {
   this->spi_setup();
 }
 void MAX31855Sensor::dump_config() {
-  LOG_SENSOR("", "MAX31855", this);
+  ESP_LOGCONFIG(TAG, "MAX31855:");
   LOG_PIN("  CS Pin: ", this->cs_);
   LOG_UPDATE_INTERVAL(this);
+  LOG_SENSOR("  ", "Thermocouple", this);
+  if (this->temperature_reference_) {
+    LOG_SENSOR("  ", "Reference", this->temperature_reference_);
+  } else {
+    ESP_LOGCONFIG(TAG, "  Reference temperature disabled.");
+  }
 }
 float MAX31855Sensor::get_setup_priority() const { return setup_priority::DATA; }
 void MAX31855Sensor::read_data_() {
   this->enable();
   delay(1);
   uint8_t data[4];
   this->read_array(data, 4);
+  this->disable();
+
+  const uint32_t mem(data[0] << 24 | data[1] << 16 | data[2] << 8 | data[3] << 0);
```

Comment:
```suggestion
  const uint32_t mem = data[0] << 24 | data[1] << 16 | data[2] << 8 | data[3] << 0;
```

I'm not sure why you used the `()` constructor syntax for declaring variables - but IMO it's much harder to read. I strongly recommend using the equals syntax.

(Applies in other places too).

Reply:
Just habit from other projects. I will happily update to match the project style later tonight.